# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1407426107 14400
#      Thu Aug 07 11:41:47 2014 -0400
# Branch stable
# Node ID ed24879107300122b97a4b86340eef70b2ece2db
# Parent  a66548dc07b0fc9b7d37a4be065aa15e8243c96a
Added tag release-3-8-2 for changeset a66548dc07b0

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -84,8 +84,9 @@ d734216aa2b1022c190e6b07d7ccdd59fe15678f
 f1fa148766d623f7f24cf541ea8b88d8c1b63d33 rc-3-8-0-2
 b72bcf5f78cc2231e9bf2735d3b8d92779fc4dff release-3-8-0
 85d31344fb23725791bc687337295d47416b9c17 rc-3-8-1-2
 02ce68d63fba31cd14ac060c087212a032ae3202 rc-3-8-1-3
 dd669c2ae76c167613f54d6f4db0130fa2124cac rc-3-8-1-4
 43cc202335dc4a53a3d8fc9ca90acaa81d2e63d3 release-3-8-1
 492c5614953571a0b07f4c1621cc7bc5c85c0251 rc-3-8-2-1
 46df2c7b8f4dec400ce353370f2e47a02926026d rc-3-8-2-2
+a66548dc07b0fc9b7d37a4be065aa15e8243c96a release-3-8-2
